<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenewPolicyForm</class>
 <widget class="QDialog" name="RenewPolicyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Renew Policy Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#RenewPolicyForm{
	background-color:  #C2272D;
}

QLabel{
	color: #fff;
}

QMessageBox QLabel {
	color: #000000;
}

#RenewPolicyForm &gt; QPushButton{
margin: 15px;
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>139</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Agreed Monthly Payment:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>143</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Payment Amount: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="paymentamountLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <double>50000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="monthlypaymentLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="monthspaidLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>139</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Months Paid</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>139</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Month Limit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="monthlimitLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: #fff;
}</string>
   </property>
   <property name="text">
    <string>Renew Policy [</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="insurancetypeLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: #fff;
}</string>
   </property>
   <property name="text">
    <string>insurance type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: #fff;
}</string>
   </property>
   <property name="text">
    <string>]</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
